careers.html

```diff
@@ -0,0 +1,143 @@
+      <li><a href="careers.html">CAREERS</a></li>
+      <li><a href="contact.html">CONTACT</a></li>
+      <li><a href="www.infratab.com" target="_blank"><i class="fa fa-globe"></i></a></li>
+    </ul>
+    <ul id="nav-mobile" class="side-nav">
+      <li><a href="index.html">HOME</a></li>
+      <li><a href="careers.html">CAREERS</a></li>
+      <li><a href="contact.html">CONTACT</a></li>
+      <li><a href="www.infratab.com"><i class="fa fa-globe"></i></a></li>
+    </ul>
+    <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">MENUs</i></a>
+  </div>
+</nav>
+<section class="team scroll">
+  <div class="team-image"></div>
+  <div class="team-overlay"></div>
+  <div class="team-center">
+      <p class="team-caption">BUILD future with us</p>
+      <a class="team-button" id="js-join-scroll">Join us</a>
+  </div>
+</section>
+
+<section class="life container">
+    <p class="section-heading">Life at <span class="blue-color">infra</span><span class="green-color">tab</span></p>
+    <p class="heading-bottom-line"></p>
+    <div class="row margin-top-10">
+        <div class="col l6">
+            <p class="life-info-heading">Growth</p>
+            <p class="life-info">We care for each other and make them feel like they belong Encourage others to participate to their fullest. Listen, communicate openly and set clear expectations and ensure that they grow rapidly while working in joyful and optimism enviornment.</p>
+        </div>
+        <div class="col l6">
+            <img src="img/teamgroup.png">
+        </div>
+    </div>
+    <div class="row margin-top-10">
+        <div class="col l6">
+            <img src="img/teamgroup.png">
+        </div>
+        <div class="col l6">
+            <p class="life-info-heading">Challenges</p>
+            <p class="life-info">We prioritize our work that advances the mission and positively impacts the team. We apply original thinking to overcome challenges and make outcome a reality.</p>
+        </div>
+    </div>
+    <div class="row margin-top-10">
+        <div class="col l6">
+            <p class="life-info-heading">Culture</p>
+            <p class="life-info">It takes an entire team united behind something big. Together, we work hard, we laugh a lot, we brainstorm nonstop, we use hundreds of Post-Its a week, and we give the best high-fives in town. Also we offer work from home to ensure the comfortess of our team members.</p>
+        </div>
+        <div class="col l6">
+            <img src="img/teamgroup.png">
+        </div>
+    </div>
+</section>
+
+<section class="join-us" id="js-join-us">
+    <p class="section-heading">Join us</p>
+    <p class="heading-bottom-line"></p>
+    <p class="section-subline">If you like getting things done, Infratab is the place for you. We are always looking for people who want to make a difference with their work.</p>
+    <div class="row">
+        <div class="col l3">
+            <div class="hiring-box">
```

Comment:
@priyanka0901 Use materializescss card instead of writing your own structure and CSS